trieste/models/keras/models.py

```diff
@@ -351,17 +351,19 @@ def __init__(
                 "batch_size": 32,
                 "callbacks": [
                     tf.keras.callbacks.EarlyStopping(
-                        monitor="loss", patience=80, restore_best_weights=True
+                        monitor="loss", patience=100, restore_best_weights=True
                     ),
                     tf.keras.callbacks.ReduceLROnPlateau(
-                        monitor="loss", factor=0.3, patience=15
+                        monitor="loss", factor=0.3, patience=20
                     )
                 ],
             }
 
         if self.optimizer.loss is None:
             self.optimizer.loss = "mse"
 
+        self._learning_rate = self.optimizer.optimizer.learning_rate.numpy()
```

Comment:
We had a very similar solution, so feel free to prioritize this one. Results will be the same.